syntax/header.php

```diff
@@ -61,12 +61,12 @@ function handle($match, $state, $pos, Doku_Handler $handler) {
 
         $this->eventhandler->notifyEvent('insert', 'header', 'header', $pos, $match, $handler);
 
-        if ($handler->status['section']) $handler->_addCall('section_close', array(), $pos);
+        if ($handler->setStatus['section']) $handler->_addCall('section_close', array(), $pos);
```

Comment:
when calling a method like setStatus you need to use round brackets. _addCall is deprecated, I believe.

Reply:
Thanks for the hints.
I have updated the _addCall as suggested in splitbrain/dokuwiki#2887